javascript/javascript3/week3/canvas/main.js

```diff
@@ -0,0 +1,67 @@
+console.log("Hello Canvas");
+
+const canvas = document.querySelector("canvas");
+const ctx = canvas.getContext("2d");
+
+canvas.height = window.innerHeight;
+canvas.width = window.innerWidth;
+
+// Class creation time!
+
+class Circle {
+  constructor(x, y, r, startAngle, endAngle, fillColor) {
+    this.x = x;
+    this.y = y;
+    this.r = r;
+    this.startAngle = startAngle;
+    this.endAngle = endAngle;
+    this.fillColor = fillColor;
+  }
+
+  draw() {
+    ctx.beginPath();
+    ctx.arc(
+      this.x,
+      this.y,
+      this.r,
+      this.startAngle,
+      this.endAngle,
+      this.fillColor
+    );
+    ctx.stroke();
+    ctx.fillStyle = this.fillColor;
+    ctx.fill();
+  }
+}
+
+const c1 = new Circle(50, 150, 20, 0, 2 * Math.PI, "#000000");
+//c1.draw();
+
+// Now lets make art!
+
+setInterval(() => {
+  const randomX = Math.floor(Math.random() * canvas.width);
```

Comment:
a function GetRandomInteger(max) would save you some typing and simplify the code